File: src/main/java/z3roco01/quickvillagers/QuickVillagers.java
package z3roco01.quickvillagers;

import net.fabricmc.api.ModInitializer;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class QuickVillagers implements ModInitializer {
    public static final String modid = "quick_villagers";
    public static final Logger logger = LoggerFactory.getLogger(modid);

    @Override
    public void onInitialize() {
        logger.info("init :3");
        QuickVillagerComponents.register();
    }
}

File: src/main/kotlin/z3roco01/quickvillagers/QuickVillagers.kt
package z3roco01.quickvillagers

import net.fabricmc.api.ModInitializer
import org.slf4j.LoggerFactory

object QuickVillagers : ModInitializer {
	val modid = "quick_villagers"
    val logger = LoggerFactory.getLogger(modid)

	override fun onInitialize() {
		logger.info("init started :3")

		logger.info("init finished !! :D")
	}
}